index.js

```diff
@@ -57,6 +80,49 @@ function assembleStyles() {
 		});
 	});
 
+	function rgb2rgb(r, g, b) {
+		return [r, g, b];
+	}
+
+	styles.color.close = '\u001b[39m';
+	styles.bgColor.close = '\u001b[49m';
+
+	styles.color.ansi = {};
+	styles.color.ansi256 = {};
+	styles.color.ansi16m = {
+		rgb: wrapAnsi16m(rgb2rgb, 0)
+	};
+
+	styles.bgColor.ansi = {};
+	styles.bgColor.ansi256 = {};
+	styles.bgColor.ansi16m = {
+		rgb: wrapAnsi16m(rgb2rgb, 10)
+	};
+
+	for (var key in colorConvert) {
+		if (!colorConvert.hasOwnProperty(key)) {
+			continue;
+		}
+
+		var fn = colorConvert[key];
+		var conversion = key.replace(/^([^2]+).+$/, '$1');
+		if (/2ansi16$/.test(key)) {
+			styles.color.ansi[conversion] = wrapAnsi16(fn, 0);
+			styles.bgColor.ansi[conversion] = wrapAnsi16(fn, 10);
+			continue;
+		}
+		if (/2ansi$/.test(key)) {
+			styles.color.ansi256[conversion] = wrapAnsi256(fn, 0);
+			styles.bgColor.ansi256[conversion] = wrapAnsi256(fn, 10);
+			continue;
+		}
```

Comment:
Ugh I'll do it tomorrow. Their rule request requirements require a requiem to [rhamnusia](http://www.godchecker.com/pantheon/greek-mythology.php?deity=NEMESIS).